### PR TITLE
[TASk] add shenyu-admin ingress and hpa

### PR DESCRIPTION
**Pull request by misaya295**
about #24

### CODE REVIEW

**Comment by JooKS-me** [comment]
Hi @aFlyBird0 , can you help have a review?

File: charts/shenyu/values.yaml [hunk 1]
```diff
@@ -8,35 +8,62 @@ admin:
...
-#      cpu: 1000m
-#      memory: 1024Mi
+  #    requests:
+  #      cpu: 100m
+  #      memory: 512Mi
+  #    limits:
+  #      cpu: 1000m
+  #      memory: 1024Mi
+  autoscaling:
+    enabled: true
```
Comment (JooKS-me) [inline charts/shenyu/values.yaml]: Let's change it to false by `default`

File: charts/shenyu/values.yaml [hunk 2]
```diff
@@ -8,35 +8,62 @@ admin:
...
+    minReplicas: 1
+    maxReplicas: 10
+    targetCPUUtilizationPercentage: 75
+    targetMemoryUtilizationPercentage: 75
+  ingress:
+    enabled: ture
```
Comment (JooKS-me) [inline charts/shenyu/values.yaml]: Let's change it to false by default

**Review by JooKS-me — CHANGES_REQUESTED** [review]

**Comment by JooKS-me** [comment]
Good job!

**Review by JooKS-me — APPROVED** [review]

**Comment by JooKS-me** [comment]
Please also add doc here: https://shenyu.apache.org/zh/helm/index/

**Comment by misaya295** [comment]
> Please also add doc here: https://shenyu.apache.org/zh/helm/index/

I will do it in next pr,which is  bootstrap hpa and ingress.

**Pull request merged**